src/main/java/baseEntities/BasePage.java

```diff
@@ -0,0 +1,15 @@
+package baseEntities;
```

Comment:
Please store BasePage class in the directory with all the pages

Reply:
done